Services/CO.CDP.EntityVerification/Api/PponEndpointExtensions.cs

```diff
@@ -9,8 +12,11 @@ public static class PponEndpointExtensions
 {
     public static void UsePponEndpoints(this WebApplication app)
     {
-        app.MapGet("/identifiers/{identifier}", (string _) => Task.FromResult(Results.NotFound()))
-            .Produces<string>(StatusCodes.Status200OK, "application/json")
+        app.MapGet("/identifiers/{identifier}",
+            async ([FromQuery] string? identifier, IUseCase<LookupIdentifierQuery, IEnumerable<Identifier>> useCase) =>
```

Comment:
It's not a query parameter, but one found in path. It should be non-nullable.

Reply:
Good spot, removed.